pkg/resources/preprocessor/workloads.go

```diff
@@ -70,31 +72,39 @@ processors:
     log_statements:
     - context: log
       statements:
-      - merge_maps(attributes, body, "upsert") where attributes["filename"] == nil
       - set(attributes["COMM"], attributes["_COMM"])
       - delete_matching_keys(attributes, "^_.*")
     - context: resource
       statements:
       - set(attributes["log_type"], "controlplane") where attributes["k8s.pod.labels.tier"] == "control-plane"
       - set(attributes["kubernetes_component"], attributes["k8s.pod.labels.component"]) where attributes["k8s.pod.labels.tier"] == "control-plane"
+
 exporters:
   opensearch:
-    endpoints: [ "{{ .Endpoint }}" ]
-    index: {{ .WriteIndex }}
+    http:
+      endpoint: {{ .Endpoint }}
+      tls:
+        ca_file: /etc/otel/chain.crt
+        cert_file: /etc/otel/certs/tls.crt
+        key_file: /etc/otel/certs/tls.key
+    logs_index: {{ .WriteIndex }}
+    dataset: kubernetes
```

Comment:
Right now, the OpenSearch trace exporter only supports indexing according to the Observability plugin [index naming conventions](https://opensearch.org/docs/latest/observing-your-data/ss4o/#traces-and-metrics), i.e., `ss4o_traces-dataset-namespace`. 

For that reason and to avoid confusion for end users, I hardcoded this index name here. 

However, we are using our own fork, so we could change those index naming conventions to use any index name that we like and allow our users to do so, as logs are done. What do you think?